### PR TITLE
Prescription window ui

### LINKED ISSUE #1956: UI: Prescription window
#### Header, Content Area, Side Panel and Footer
- Use the `Outbound Shipment` as a base
    - Remove Service Charge from Side Panel (?)
    - Remove Transport Details from Side Panel 
- Statuses for `Prescriptions` are: New, Picked and Verified. 
- The header should contain: 
    - Patient name (can search by code or name)
    - Replace customer ref with `Clinician Name`: can search by code or name (?)
- Items should be grouped automatically.

### Additional context 💌
dispensary-detail-view

### Moneyworks Jobcode 🧰
OMS:MVP | Open mSupply MVP Prescription Module
-- | --

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #1956 See #2039 first for most translations and hooks

# 👩🏻‍💻 What does this PR do? 
The prescription detail view. Currently the `Add Item` and line row click does nothing. Current Prescription view allows you to delete lines and change the following:

- Patient
- Clinician
- Statuses
- Colour
- Note

Haven't added the `Print` button yet since need to decide on how the labels should look

# 🧪 How has/should this change been tested? 
- [ ] Create some prescriptions in mSupply
- [ ] Transfer store back to remote site
- [ ] Sync
- [ ] Click on a prescription
- [ ] See

### CODE REVIEW

File: client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  Grid,
+  useTranslation,
+  useIntlUtils,
+  DropdownMenu,
+  DropdownMenuItem,
+  DeleteIcon,
+} from '@openmsupply-client/common';
+import { PatientSearchInput } from '@openmsupply-client/system';
+import { usePrescription } from '../api';
+import { ClinicianSearchInput } from 'packages/system/src/Clinician';
```
Comment (mark-prins) [inline client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx:15]:
```suggestion
import { ClinicianSearchInput } from '../../../../system/src/Clinician';
```

File: server/repository/src/db_diesel/activity_log_row.rs [hunk 2]
```diff
@@ -50,6 +50,8 @@ pub enum ActivityLogType {
     StockOnHold,
     StockOffHold,
     Repack,
+    PrescriptionCreated,
+    PrescriptionDeleted,
```
Comment (mark-prins) [inline server/repository/src/db_diesel/activity_log_row.rs:54]: perhaps also `PrescriptionStatusVerified` and `PrescriptionStatusPicked` ?

image

File: server/repository/src/migrations/v1_02_01/mod.rs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+use super::{version::Version, Migration};
+
+use crate::StorageConnection;
+pub(crate) struct V1_02_01;
+
+impl Migration for V1_02_01 {
```
Comment (mark-prins) [inline server/repository/src/migrations/v1_02_01/mod.rs:6]: the migrations need to be added to `/server/repository/src/migrations/mod.rs` too 🙏

**Review by mark-prins — CHANGES_REQUESTED** [review]
Working nicely! Just the migrations which need to be added. The other items have already been addressed 🙏 

one extra request - can you change the first line in `/client/packages/invoices/src/index.ts` to 
`import { ItemRowFragment } from '../../system/src';` please?

File: server/repository/src/migrations/v1_02_01/mod.rs (hunk 3 above)
Comment (roxy-dao) [reply server/repository/src/migrations/v1_02_01/mod.rs:7]: [20c0cd1](https://github.com/openmsupply/open-msupply/pull/2038/commits/20c0cd1c18240acbf8cd92b698cad41de916ffc2)

**Pull request merged**